shared/clinical-code-sets/medications/serotonin-modulator-stimulator/1/README.md

```diff
@@ -0,0 +1,15 @@
+# Serotonin Modulator and Stimulator (SMS)  
+
+This code set was created from getset.ga, using the following list from the PI of RQ051:
+-- Vortioxetine hydrobromide
```

Comment:
According to BNF this is sold under the name Brintellix - which does have read v2 and CTV3 codes, so these should be added.